core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -367,6 +367,28 @@
  * <td>Gauge</td>
  * <td></td>
  * </tr>
+ * <!-- scan server -->
+ * <tr>
+ * <th>N/A</th>
+ * <th>N/A</th>
+ * <th>{@value #METRICS_SSERVER_REGISTRATION_TIMER}</th>
```

Comment:
There is a mismatch between this code and this name.  The ScanServer code uses the terminology of reserving files.  The metric names should probably reflect that and use reservation in their names instead of registration.

```suggestion
 * <th>{@value #METRICS_SSERVER_RESERVATION_TIMER}</th>
```

Reply:
Changed in 5dd3c2d311